src/arrays.js

```diff
@@ -1,62 +1,106 @@
 const getNthElement = (index, array) => {
-  // your code here
+  let myIndex = index % array.length;
+  return array[myIndex];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  let text = array.join();
+  return text;
 };
 
 const csvStringToArray = string => {
-  // your code here
+  let text = string.split(',');
+  return text;
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
+//  return array; - I got stuck here, discussed with Jasmin 24/04 and the test is expecting undefined :)
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  // test states we need to return new array
+  let newArray = [];
+  for (let i = 0; i < array.length; i++) {
+    newArray.push(array[i]);
+  }
+  newArray.push(element);
+  return newArray;
```

Comment:
Can be simplified by writing `return array.concat(element);`, [here's a link to the mdn docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) if you would like to read more about concat